packages/scandipwa/src/component/MyAccountMyWishlist/MyAccountMyWishlist.container.js

```diff
@@ -102,7 +102,20 @@ export class MyAccountMyWishlistContainer extends PureComponent {
             /** @namespace Component/MyAccountMyWishlist/Container/moveWishlistToCartThen */
             () => this.showNotificationAndRemoveLoading('Available items moved to cart'),
             /** @namespace Component/MyAccountMyWishlist/Container/moveWishlistToCartCatch */
-            (error) => this.showErrorAndRemoveLoading(getErrorMessage(error))
```

Comment:
Maybe better to move those changes in an original method?

Otherwise, `showErrorAndRemoveLoading` is not used in this class at all. But if you remove it, we'll kinda lose extensibility for the function.